src/ray/gcs/redis_module/ray_redis_module.cc

```diff
@@ -192,7 +192,7 @@ flatbuffers::Offset<flatbuffers::String> RedisStringToFlatbuf(
 /// \return OK if there is no error during a publish.
 int PublishTableUpdate(RedisModuleCtx *ctx, RedisModuleString *pubsub_channel_str,
                        RedisModuleString *id, GcsTableNotificationMode notification_mode,
-                       RedisModuleString *data) {
+                       RedisModuleString *data, bool reply_if_ok) {
```

Comment:
This flag is awkward. Can we remove it and just have this function always return `REDISMODULE_OK`? I would prefer if we just had the caller explicitly check the value and call `RedisModule_ReplyWithSimpleString(ctx, "OK")` if necessary. We can make the pattern a macro if it becomes cumbersome.

Reply:
I remove the flag and change the function return value to `Status`. Then we can call the reply function at the top layer.